insights/parsers/softnet_stat.py

```diff
@@ -0,0 +1,104 @@
+"""
+softnet_stat - /proc/net/softnet_stat
+=====================================
+
+This parser parses the stats from network devices. These stats includes events
+per cpu(in row), number of packets processed i.e packet_process (first column),
+number of packet drops packet_drops (second column), time squeeze i.e net_rx_action
+performed time_squeeze(third column), cpu collision i.e collision occur while obtaining
+device lock while transmitting cpu_collision packets (Ninth column), received_rps
+number of times cpu woken up received_rps (tenth column), number of times reached flow
+limit count flow_limit_count (eleventh column).
```

Comment:
I would strongly recommend you add something here about how these stats are somewhat undocumented and may change - e.g. from https://blog.packagecloud.io/eng/2016/06/22/monitoring-tuning-linux-networking-stack-receiving-data/#procnetsoftnetstat:

> These statistics are output to a file in proc: /proc/net/softnet_stat for which there is, unfortunately, very little documentation. The fields in the file in proc are not labeled and could change between kernel releases.

I agree that it's unlikely that the positions of those values will change in the short term, so I don't think it's worth (at this stage) trying to work out which version of the kernel you're extracting data from and trying to cope with field source changes over time.  But it'd be good to have a caveat there.